src/sql/workbench/browser/modelComponents/card.component.html

```diff
@@ -1,60 +1,66 @@
-<div #cardDiv role="radio" *ngIf="label" [class]="getClass()" (click)="onCardClick()" [attr.aria-checked]="selected" (mouseover)="onCardHoverChanged($event)"
-	(mouseout)="onCardHoverChanged($event)" tabIndex="0" [style.width]="width" [style.height]="height">
-	<ng-container *ngIf="isVerticalButton || isDetailsCard">
-		<span *ngIf="hasStatus" class="card-status">
-			<div class="status-content" [style.backgroundColor]="statusColor"></div>
-		</span>
-		<span *ngIf="showRadioButton" class="selection-indicator-container">
-			<div *ngIf="showAsSelected" class="selection-indicator"></div>
-		</span>
-		<ng-container *ngIf="isVerticalButton">
-			<div class="card-vertical-button">
-				<div *ngIf="iconPath" class="iconContainer">
-					<div [class]="iconClass" [style.width]="iconWidth" [style.height]="iconHeight"></div>
-				</div>
-				<h4 class="card-label">{{label}}</h4>
-				<div *ngIf="descriptions.length > 0" class="model-card-description-container">
-					<div *ngFor="let desc of descriptions">
-						<div *ngIf="desc.label; else separator" [style.font-weight]="desc.fontWeight" class="model-card-list-item-description">
-							<span>{{desc.label}}</span><span class="model-card-list-item-description-value">{{desc.value}}</span>
+<!-- The horizontal is for compatibility, we're going to get rid of this once card component can be removed -->
+<div class="horizontal">
```

Comment:
Why wrap it in another div? Why not just add the class to the existing #cardDiv instead? Not a huge deal but less indentation and text is usually nice